test/rx/components/Prescription.unit.spec.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+import { expect } from 'chai';
+
+import Prescription from '../../../src/js/rx/components/Prescription.jsx';
+import { prescriptions } from '../../util/rx-helpers.js';
+
+const item = prescriptions.data[0];
+
+const props = {
+  ...item,
+  glossaryModalHandler: () => {},
+  refillModalHandler: () => {},
+  dispatch: () => {}
+};
+
+describe('<Prescription>', () => {
+  it('should render', () => {
+    const tree = SkinDeep.shallowRender(<Prescription {...props}/>);
+    const vdom = tree.getRenderOutput();
+    expect(vdom).to.not.be.undefined;
+  });
+
+  it('should render tracking link if applicable', () => {
+    const newProps = {
+      ...props,
+      attributes: {
+        ...props.attributes,
+        isTrackable: true,
+      },
+    };
+
+    const tree = SkinDeep.shallowRender(<Prescription {...newProps}/>);
+    expect(tree.subTree('TrackPackageLink')).to.exist;
```

Comment:
yep this is tricky because `undefined !== false` so one of my tests was passing even though it shouldn't have been